x-pack/filebeat/module/azure/auditlogs/config/auditlogs.yml

```diff
@@ -7,3 +9,15 @@ expand_event_list_from_field: "records"
 username: "$ConnectionString"
 password: {{ .connection_string }}
 ssl.enabled: true
+
+{{ else if eq .input "file" }}
```

Comment:
maybe separate this part into a separate `file.yml` file? Like this: https://github.com/elastic/beats/tree/master/x-pack/filebeat/module/aws/s3access/config